File: Projet Java/PidevMaster/src/Entities/Activite.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

/**
 *
 * @author sa3do
 */
public class Activite {
    private int IdActivite;
    private String NomActivite;
    private String Description;
    private String Type;

    public Activite(String NomActivite, String Description, String Type) {
        this.NomActivite = NomActivite;
        this.Description = Description;
        this.Type = Type;
    }

    public Activite() {
       
    }

    public int getIdActivite() {
        return IdActivite;
    }

    public void setIdActivite(int IdActivite) {
        this.IdActivite = IdActivite;
    }

    public String getNomActivite() {
        return NomActivite;
    }

    public void setNomActivite(String NomActivite) {
        this.NomActivite = NomActivite;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public String getType() {
        return Type;
    }

    public void setType(String Type) {
        this.Type = Type;
    }

    @Override
    public String toString() {
        return "\nActivite{" + "IdActivite=" + IdActivite + ", NomActivite=" + NomActivite + ", Description=" + Description + ", Type=" + Type + '}';
    }
    
    
    
}

File: Pidev/src/ProjetBundle/Entity/Activite.php
<?php

namespace ProjetBundle\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;

/**
 * Activite
 *
 * @ORM\Table(name="activite")
 * @ORM\Entity
 */
class Activite
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */

    private $idactivite;

    /**
     * @var string
     *
     * @ORM\Column(name="NomActivite", type="string", length=100, nullable=false)
     */
    private $nomactivite;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="text", length=65535, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="Type", type="string", length=100, nullable=false)
     */
    private $type;
   /* /**
     * @ORM\ManyToMany(targetEntity="Classe",mappedBy="Activite")
     */
   /* private $classe;
    public function __construct()
    {
        $this->classe = new ArrayCollection();
    }*/

    /**
     * @return int
     */
    public function getIdactivite()
    {
        return $this->idactivite;
    }

    /**
     * @param int $idactivite
     */
    public function setIdactivite($idactivite)
    {
        $this->idactivite = $idactivite;
    }

    /**
     * @return string
     */
    public function getNomactivite()
    {
        return $this->nomactivite;
    }

    /**
     * @param string $nomactivite
     */
    public function setNomactivite($nomactivite)
    {
        $this->nomactivite = $nomactivite;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }

    /**
     * @return ArrayCollection
     */
    public function getClasse()
    {
        return $this->classe;
    }

    /**
     * @param ArrayCollection $classe
     */
    public function setClasse($classe)
    {
        $this->classe = $classe;
    }



}

